packages/blockchain-wallet-v4-frontend/src/data/rootReducer.ts

```diff
@@ -9,7 +9,7 @@ import cache from './cache/reducers'
 import components from './components/reducers'
 import { custodialReducer as custodial } from './custodial/reducers'
 import form from './form/reducers'
-import goals from './goals/reducers'
+import { reducer as goals } from './goals/slice'
```

Comment:
this should be `goalsReducer`, just like the rest here

Reply:
fixed!